### ISSUE TITLE
Missing files in npm package

### ISSUE THREAD

**Issue by wolfy1339** [wheresrhys/fetch-mock#678]
Hi,

It seems that installs of `10.x` of `fetch-mock` are missing some files.
```json
  "main": "./cjs/server.js",
  "browser": "./esm/client.js",
  "module": "./esm/server.js",
  "types": "./types/index.d.ts",
```

Those files referenced are not present in the npm package. In the package there is only a `types` folder and `dist` folder that only contains the CJS release

I'm trying to update to the new release instead of using the fork from #644 and this is currently preventing it's use

**Comment by wheresrhys** [comment]
Thanks for raising this. I'll have a look at the build step

**Comment by wheresrhys** [comment]
I think this is now fixed in 10.0.5
